pages/dkp/kaptain/2.1.0/tutorials/ngc/index.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: layout.pug
+navigationTitle: NGC catalog
+title: NVIDIA NGC catalog
+menuWeight: 30
+excerpt: Use NVIDIA's NGC catalog with Kaptain
+beta: false
+enterprise: false
+---
+
+Kaptain supports the usage of Nvidia's NGC catalog in networked environments, which you can access and set up via the Kubeflow UI. The NGC catalog includes containers, charts, pre-trained models, toolkits and more, which are optimized for GPU deployment and can now be used to run AI workflows in Kaptain. You can find all the available resources for your AI/ML workflow in the [NVIDIA NGC catalog][NGC_catalog].
+
+## Prerequisites
+
+-   You have [deployed Kaptain into a cluster](../../install/).
+
+-   You have a Kubeflow profile, and [permissions to create notebooks](../../user-management/).
+
+-   You have [installed JupyterLab](https://jupyterlab.readthedocs.io/en/2.2.x/getting_started/installation.html).
+
+-   You have permissions to access the [NGC component](https://docs.nvidia.com/ngc/ngc-catalog-user-guide/index.html#registering-activating-ngc-account).
+
+## Prepare your Kubeflow user profile
+
+You do not need to download, nor install the NGC catalog. You will be able to access it by logging into your Kaptain instance. However, you have to set the default entry point for Jupyter Lab first.
+
+In your Kubeflow profile, create a custom configuration for a notebook server to launch in Jupyter Lab. Specify the `user-namespace` where you want to create the notebook.
+
+1.  Open the `PodDefault` YAML file:
+
+    <!-- TODO: command to "open" or "edit" the yaml -->
+
+    ```yaml
+    apiVersion: "kubeflow.org/v1alpha1"
+    kind: PodDefault
+    metadata:
+    name: custom-entrypoint
+    namespace: <user-namespace>
+    spec:
+    selector:
+        matchLabels:
+        custom-entrypoint: "true"
+    desc: "Launch as Jupyter Lab"
+    command:
+    - jupyter
+    args:
+    - lab
+    - --notebook-dir=/workspace
+    - --ip=0.0.0.0
+    - --no-browser
+    - --allow-root
+    - --port=8888
+    - --NotebookApp.token=''
+    - --NotebookApp.password=''
+    - --NotebookApp.allow_origin='*'
+    - --NotebookApp.base_url=$(NB_PREFIX)
+    ```
+
+1.  Run this command to apply the new entry point:
+
+    ```bash
+    k apply -n user -f pd.yaml
```

Comment:
They wouldn't need to specify the namespace (-n ) as it is in the .yaml file.    Also, we should not use aliases or shortcuts in commands like this.  'k' is an alias that many developers use to avoid typing 'kubectl', but for completeness we should spell it out.

```suggestion
    kubectl apply -f podddefault.yaml
```